packages/tools/devtools/devtools-core/src/data-visualization/SharedTreeVisualizer.ts

```diff
@@ -78,18 +99,44 @@ function createToolTipContents(schema: SharedTreeSchemaNode): VisualTreeNode {
 }
 
 /**
- * Constructs a VisualTree of the input tree's schema fields in {@link VisualTreeNode} or {@link VisualValueNode}.
+ * Converts the visual representation from {@link visualizeSharedTreeNodeBySchema} to a visual tree compatible with the devtools-view.
+ * @param tree - the visual representation of the SharedTree.
+ * @returns - the visual representation of type {@link VisualChildNode}
```

Comment:
Note: `@returns` comments don't want a `-` between the tag and the docs. `@param` comments are the only ones that use this pattern.